### PR TITLE
Update extra_tests YAML schedule for opensuse

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/71032

openqa-clone-custom-git-refspec https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11090 https://openqa.opensuse.org/tests/1405831 YAML_SCHEDULE=schedule/functional/extra_tests_textmode.yaml
Created job #1406423: opensuse-Tumbleweed-DVD-x86_64-Build20200924-extra_tests_in_textmode@64bit -> https://openqa.opensuse.org/tests/1406479

### CODE REVIEW

File: schedule/functional/extra_tests_textmode.yaml [hunk 1]
```diff
@@ -80,4 +128,5 @@ schedule:
     - console/aaa_base
     - console/osinfo_db
     - console/zypper_ref
```
Comment (foursixnine) [inline schedule/functional/extra_tests_textmode.yaml:130]: zypper_ref here, would be redundant now

File: schedule/functional/extra_tests_textmode.yaml [hunk 2]
```diff
@@ -17,16 +17,64 @@ conditional_schedule:
                 - console/yast2_lan_device_settings
             x86_64:
                 - console/yast2_lan_device_settings
+    zypper:
+        DISTRI:
+            opensuse:
```
Comment (foursixnine) [inline schedule/functional/extra_tests_textmode.yaml:22]: wouldn't this mean that these test would only be executed for opensuse? I guess we don't really need them on sles, or do we?

File: schedule/functional/extra_tests_textmode.yaml [hunk 3]
```diff
@@ -17,16 +17,64 @@ conditional_schedule:
...
+                - update/zypper_clear_repos
+                - console/zypper_ar
+                - console/zypper_ref
+                - console/zypper_ar
```
Comment (foursixnine) [inline schedule/functional/extra_tests_textmode.yaml:26]: I guess this is extra.

File: schedule/functional/extra_tests_textmode.yaml [hunk 4]
```diff
@@ -17,16 +17,64 @@ conditional_schedule:
...
+    sle_tests:
+        DISTRI:
+            sle:
+                - console/supportutils
 schedule:
     - installation/bootloader_start
     - boot/boot_to_desktop
     - console/prepare_test_data
     - console/consoletest_setup
+    - '{{zypper}}'
```
Comment (foursixnine) [inline schedule/functional/extra_tests_textmode.yaml:72]: I'd rename this as `updaterepos` or something like this

**Review by foursixnine — CHANGES_REQUESTED** [review]

File: schedule/functional/extra_tests_textmode.yaml (hunk 1 above)
Comment (asdil12) [reply schedule/functional/extra_tests_textmode.yaml:130]: zypper_ref is executed multiple times in the original ´extra_tests_in_textmode`

File: schedule/functional/extra_tests_textmode.yaml (hunk 3 above)
Comment (asdil12) [reply schedule/functional/extra_tests_textmode.yaml:26]: extra?

File: schedule/functional/extra_tests_textmode.yaml (hunk 2 above)
Comment (asdil12) [reply schedule/functional/extra_tests_textmode.yaml:22]: I tried to resemble what the original schedule was doing.

**Comment by asdil12** [comment]
openqa-clone-custom-git-refspec https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11090 https://openqa.opensuse.org/tests/1410476 YAML_SCHEDULE=schedule/functional/extra_tests_textmode.yaml
Created job #1412836: opensuse-Tumbleweed-DVD-x86_64-Build20200927-extra_tests_in_textmode@64bit -> https://openqa.opensuse.org/t1412836

**Review by foursixnine — APPROVED** [review]

**Pull request merged**